extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -192,7 +195,7 @@ describes.realWin('amp-bind', {
   });
 
   it('should verify string attribute bindings in dev mode', () => {
-    env.sandbox.stub(window, 'AMP_MODE', {development: true});
+    env.sandbox.stub(getMode(), 'development', true);
```

Comment:
Is there a reason why that's better? I changed this because `AMP_MODE` is technically an implementation detail of `getMode`.